nextflow.config

```diff
@@ -1,20 +1,21 @@
 process {
-    queue="$SCXA_LSF_QUEUE"
-    clusterOptions="$SCXA_LSF_OPTIONS"
-    executor='lsf'
-    
-    withName: 'download_fastqs' {
-        clusterOptions = "-g /scxa_ena_download"
-    }
-}
-
-executor {
+    executor='slurm'
+    queue="$SCXA_HPC_QUEUE"
```

Comment:
A small suggestion: Instead of `queue` we can use `partition` since that's the equivalent in SLURM.

Reply:
@irisdianauy  We can't, the process directive `queue` is fixed by Nextflow https://www.nextflow.io/docs/latest/executor.html#slurm